libnavigation-base/src/main/java/com/mapbox/navigation/base/typedef/RoundingIncrement.kt

```diff
@@ -8,6 +8,9 @@ const val ROUNDING_INCREMENT_TWENTY_FIVE = 25
 const val ROUNDING_INCREMENT_FIFTY = 50
 const val ROUNDING_INCREMENT_ONE_HUNDRED = 100
 
+/**
+ * Defines threshold in meters for determination is user off the route
```

Comment:
See above about `roundingIncrement`.

Reply:
Are you wondering if we should move this `annotation class` to `libnavigation-core`? If so, yes definitely can be moved there cause as you mentioned https://github.com/mapbox/mapbox-navigation-android/pull/2447/files#r378345039 at the moment is only accessed by `MapboxDistanceFormatter` which lives in `libnavigation-core` module. cc @abhishek1508